sklearn/preprocessing/data.py

```diff
@@ -1708,49 +1716,56 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
 
         Non-categorical features are always stacked to the right of the matrix.
 
-    dtype : number type, default=np.float
+    dtype : number type, default=np.float64
         Desired dtype of output.
 
     sparse : boolean, default=True
         Will return sparse matrix if set True else will return an array.
 
     handle_unknown : str, 'error' or 'ignore'
-        Whether to raise an error or ignore if a unknown categorical feature is
-        present during transform.
+        Whether to raise an error or ignore if an unknown categorical
+        feature is present during transform.
 
     Attributes
     ----------
+    feature_index_range_ : array, shape [n_feature, 2]
+        ``feature_index_range_[i]`` specifies the range of column indices
+        occupied by the input feature `i` in the one-hot encoded array.
+
+    one_hot_feature_index_ : array, shape [n_features_new]
```

Comment:
Why do we need this as well as `feature_index_range`?

Don't we also need an attribute which indicates what values correspond to each feature of the output?